src/js/common/components/Search/SearchBar2024.jsx

```diff
@@ -58,8 +59,10 @@ class SearchBar2024 extends Component {
     }
   }
 
-  
- handleKeyPress = () => {
+ handleSearchBarKeyPress = () => {
+  const { location: { pathname: currentPathname } } = window;
+  const page = lookupPageNameAndPageTypeDict(currentPathname);
```

Comment:
I preferred the variable name "currentPage" (as opposed to "page") from your prior pull request. Should stick to the same standard.